src/main/java/com/knu/buga1chuk/service/PersonService.java

```diff
@@ -4,25 +4,41 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Random;
 
 public class PersonService {
```

Comment:
Great solution!!!

I have one concern. 
Hard-coded values. It is not a good approach. It would be better to extract them into separate... FILE!!
I would be config file. Config file for our generator.